### DISCUSSION TITLE
How to set up Open Metadata using Docker and AWS ECS

### DISCUSSION THREAD

**Discussion by K-Okad** [open-metadata/OpenMetadata#19164] [Q&A]
I checked the reference link below, but I couldn't find the location of the env-mysql file mentioned in https://docs.open-metadata.org/latest/deployment/docker#4.-start-the-docker-compose-services. I'm not sure how to proceed from here. If anyone knows or has experience setting this up on ECS, please let me know.

**Comment by Prajwal214** [comment]
You need to create env-mysql with your db values as mentioned in the docs, 
[https://docs.open-metadata.org/latest/deployment/](https://docs.open-metadata.org/latest/deployment/docker#3.-update-environment-variables-required-for-openmetadata-dependencies)...- https://github.com/open-metadata/OpenMetadata/blob/main/docker/docker-compose-ingestion/env-mysql

**Comment by K-Okad** [comment]
@Prajwal214 
Thank you for reply, and I have created env-mysql with my db values as mentioned in the docs,
and I tried to docker compose it again,
but The execute migrate all 's container returned an Exit 1 code, and upon checking the Docker logs, it appears that a Java error has occurred. 
<img width="886" alt="cap1" src="https://github.com/user-attachments/assets/b1568535-7619-4262-a1dc-03d9ed2b5f19" />
<img width="882" alt="cap2" src="https://github.com/user-attachments/assets/e847fc58-2994-4ac3-886a-3ad585d47a95" />
